spec/features/access_indexing_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# NOTE: this spec is intended for use testing changes to dor-services-app mappings to cocina and
+#  testing changes to dor_indexing_app to index access from cocina
+#  to run:  bundle exec rspec --tag indexing spec/features/access_indexing_spec.rb
+RSpec.describe 'Argo rights changes result in correct Access Rights facet value', type: :feature, indexing: true do
+  let(:random_word) { RandomWord.phrases.next }
+  let(:object_label) { "Object Label for #{random_word}" }
+  let(:start_url) { "#{Settings.argo_url}/registration" }
+  let(:source_id) { "access-rights-test:#{random_word}" }
+
+  before do
+    authenticate!(start_url: start_url,
+                  expected_text: 'Register DOR Items')
+  end
+
+  scenario do
+    # fill in registration form
+    select 'integration-testing', from: 'Admin Policy'
```

Comment:
I think this breaks if we don't have this Admin Policy or there are two of them with this name.

Reply:
most of our integration tests use this admin policy, and in fact, I just copied this part from another test.

Reply:
Ah, looks like many of our other tests have the same problem.

Reply:
so ... would you care to approve this PR?   asking for a friend.